src/markdown-pages/build-apps/add-nerdgraphquery-guide.mdx

```diff
@@ -29,19 +29,19 @@ We also have a 14-minute video that covers the steps below.
 
 ## Before you begin
 
-To develop projects, you need our New Relic One CLI (command line interface).
+To develop projects, you need our New Relic One CLI (command line interface). If you haven't already installed it, do the following:
 
-If you haven't already installed it, do the following:
-
-* Go to our [installation wizard](https://one.newrelic.com/launcher/developer-center.launcher?pane=eyJuZXJkbGV0SWQiOiJkZXZlbG9wZXItY2VudGVyLmRldmVsb3Blci1jZW50ZXIifQ==) and complete steps 1–4.
-* Make a copy of your account ID in Step 1 because you’ll need it later (it's the number before the account name).
+* Install [Node.js](https://nodejs.org/en/download/).
+* Complete steps 1–4 of our [CLI wizard](https://one.newrelic.com/launcher/developer-center.launcher?pane=eyJuZXJkbGV0SWQiOiJkZXZlbG9wZXItY2VudGVyLmRldmVsb3Blci1jZW50ZXIifQ==), and be sure to make a copy of your account ID from step 1 because you’ll need it later.
 
 <Important>
 
   If you already installed the New Relic One CLI, but you can't remember your account ID, you can re-run the wizard just to get your account ID.
 
 </Important>
 
+For additional details, see [Set up your development environment](https://developer.newrelic.com/build-apps/set-up-dev-env).
```

Comment:
@rhetoric101 - should be: see [Set up your development environment](/build-apps/set-up-dev-env).

(note - link is just (/build-apps/set-up-dev-env) - GH is transforming the relative path so be careful)

Reply:
To make this relative, I think we just need "set-up-dev-env" since it's in the same directory. I pushed up this change.